prov/efa/src/efa_domain.c

```diff
@@ -171,11 +172,16 @@ static int efa_check_fork_enabled(struct fid_domain *domain_fid)
 
 #endif /* HAVE_IBV_IS_FORK_INITIALIZED */
 
-	buf = malloc(ofi_get_page_size());
+	page_size = ofi_get_page_size();
+	if (page_size <= 0) {
+		return page_size;
```

Comment:
Is it appropriate to return page_size here? e.g., if page_size somehow is returned as 0, that could be interpreted as fork support disabled, which is incorrect.

Reply:
I'll think about it